library/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # ########################################################################
-# Copyright 2016-2021 Advanced Micro Devices, Inc.
+# Copyright 2016-2022 Advanced Micro Devices, Inc.
```

Comment:
Should we maybe update the copyright date in `LICENSE.md` too?

Reply:
Didn't actually intend to make that change, had it in the stashed commit. The other libraries are still at 2021, so for consistency we could revert this, but updating the copyright dates also seems reasonable. Preferred solution?

Reply:
If you modify a file in the current year, then you update the copyright. That is the policy we adopted, and I think the clang format script takes care of applying it.

Reply:
> The other libraries are still at 2021

The main reason I suggested this is that hipSOLVER's LICENSE.md still just says 2020.

Reply:
> If you modify a file in the current year, then you update the copyright.

So should we leave the copyright date in `LICENSE.md` at 2020, as that file wasn't changed?

Reply:
No. Unlike the copyright notices in other files, `LICENSE.md` is supposed to reflect the whole repository. It only says 2020 because we forgot to update it. It should probably say 2020-2021 for ROCm 5.0 and 2020-2022 for ROCm 5.1.